### ISSUE TITLE
Opened message box cannot be navigated using VoiceOver shortkey

### ISSUE THREAD

**Issue by JasonYeMSFT** [electron/electron#13304]
* Electron Version: 2.0.0
* Operating System (Platform and Version): Mac OS
* Last known working Electron version: N/A

**Expected Behavior**
When an electron native message box is opened with a parent browser window, VoiceOver will announce the current element in focus and announce the shortkey to navigate elements in the message box. User should be able to do that.

**Actual behavior**
VoiceOver announces the current element and the shortkey to navigate. However, if the user presses that shortkey, the focus goes back to the parent browser window.

**To Reproduce**

- Clone [electron-quick-start](https://github.com/JasonYeMSFT/electron-quick-start)
- Launch the app
- Turn on VoiceOver
- Click "dialog" button in the main window
- Press the shortkey announced by VoiceOver

**Comment by sofianguy** [comment]
Thank you for taking the time to report this issue and helping to make Electron better.

The version of Electron you reported this on has been superseded by newer releases. 

If you're still experiencing this issue in Electron v4.2.x or later, please add a comment specifying the version you're testing with and any other new information that a maintainer trying to reproduce the issue should know.

I'm setting the `blocked/need-info` label for the above reasons. This issue will be closed 7 days from now if there is no response.

Thanks in advance! Your help is appreciated.

**Comment by sofianguy** [comment]
Thank you for your issue!

We haven't gotten a response to our questions in our comment above. With only the information that is currently in the issue, we don't have enough information to take action. I'm going to close this but don't hesitate to reach out if you have or find the answers we need, we'll be happy to reopen the issue.
